src/actions/handleGithubComment.js

```diff
@@ -0,0 +1,50 @@
+
+const axios = require('axios');
+const config = require('../../.config')
+module.exports = ({task, conn}) => async function handleGithubComment(params) {
+    const {Subscriber} = conn.models;
+
+    const {comment, issue} = params;
+    const orgs = await axios.get(comment.user['organizations_url']).then(res => res.data);
```

Comment:
Let's just remove this and rely on `githubOrganizationMembers.login` + `githubOrganizationMembers.id` on L17 and L18. Remove `{ githubOrganization: { $in: orgNames } }` and `{ githubOrganizationId: { $in: orgIds } },` from the `$or` below